src/commands/install-packages.yml

```diff
@@ -0,0 +1,77 @@
+description: >
+  Install your composer packages with automated caching and best practices applied.
+
+parameters:
+  app-dir:
+    type: string
+    default: ~/project
+    description: Path to the directory containing your composer.json file. Not needed if composer.json lives in the root.
+  install-command:
+    type: string
+    default: composer install --no-interaction --prefer-dist
+    description: |
+      By default, packages will be installed with "composer install --no-interaction --prefer-dist", use this to override the standard "install" command.
+  with-cache:
+    type: boolean
+    default: true
+    description: Enable automatic caching of your dependencies for increased speed.
+  cache-version:
+    type: string
+    default: v1
+    description: Change the default cache version if you need to clear the cache for any reason.
+  cache-key:
+    type: string
+    default: composer.lock
+    description: If this file is updated a new cache bucket will be created. Recommended to use composer.lock. Use composer.json when composer.lock is absent.
+  include-branch-in-cache-key:
+    type: boolean
+    default: true
+    description: >
+      If true, this cache bucket will only apply to jobs within the same branch.
```

Comment:
I would probably opt to drop this in favour of a Composer.lock checksum. I can see the advantage to more a more flexible approach, but it may not be utilized very often, and the composer.lock file might offer better cache reuse.

Reply:
Agree to drop `include-branch-in-cache-key`. 

How about the `cache-key`?
It is not unusual that composer package authors git ignores `composer.lock`.

Reply:
Removed `include-branch-in-cache-key` in https://github.com/CircleCI-Public/php-orb/pull/25/commits/c3ad40bc2529802d93733089e5b789bbe06ac3b2